.github/workflows/integration-test.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: Install Dependencies
         shell: bash
         run: |
-          sudo apt update && sudo apt install -y --no-install-recommends build-essential make libncurses5
+          sudo apt update && sudo apt install -y --no-install-recommends build-essential make
```

Comment:
@tklauser I wonder if the integration tests are failing because of this change :thinking:

Reply:
I've dropped #35408 from this PR for now and marked it as `backport/author` so we can figure out a solution, also see https://github.com/cilium/cilium/pull/35408#issuecomment-2431288124